systemverilog-plugin/UhdmAst.cc

```diff
@@ -1633,6 +1644,7 @@ void UhdmAst::process_design()
         visitEachDescendant(current_node, [&](AST::AstNode *node) {
             node->attributes.erase(UhdmAst::packed_ranges());
             node->attributes.erase(UhdmAst::unpacked_ranges());
+            node->attributes.erase(UhdmAst::is_simplified_wire());
```

Comment:
Memory leak. Attribute nodes have to be `delete`d. `erase` just removes pointer from the map.